docs/feature-roles.asciidoc

```diff
@@ -368,6 +369,77 @@ need to view monitoring data for {beatname_uc}:
 |Grants access to monitoring indices for {beatname_uc}
 |====
 
+////
+***********************************  ***********************************
+***********************************  ***********************************
+////
+
+[[privileges-api-key]]
+=== Grant privileges and roles needed for API key management
+
+++++
+<titleabbrev>Create an _API key_ user</titleabbrev>
+++++
+
+You can configure <<api-key,API keys>> to authorize requests to APM Server.
+To create an APM Server user with the required privileges for creating and managing API keys:
+
+. Create an **API key role**, called something like `apm_api_key`,
+that has the following `cluster` level privileges:
++
+[options="header"]
+|====
+| Privilege | Purpose
+
+|`manage_api_key`
+|Allow {beatname_uc} to create, retrieve, and invalidate API keys
+|====
+
+. Depending on what the **API key role** will be used for,
+also assign any or all of the following `apm` application level privileges:
++
+* To **manage Agent configuration**, assign `config_agent:read`.
```

Comment:
```suggestion
* To **receiving Agent configuration**, assign `config_agent:read`.
```

This is for agents to poll/receive agent config, not for end users to manage the config.

Reply:
Good call! I'm going to change this again from `receiving` to `receive`.